### ISSUE TITLE
To improve for assignment 3

### ISSUE THREAD

**Issue by sivert-io** [SivertGullbergHansen/IDG_gc_1007_gc_12_GuardiansOfCSS#62]
To improve for Assignment 3:

- [x] You can improve the CSS Architecture, for example you can move Animtion folder to the Style folder

- [x] Import SVGs from files, don't add long SVGs to the index.html

To improve in reflection:

- [x] Could you explain use of WebSockets in the project?

- [x] How are you using jekyll-theme-cayman?

**Comment by sivert-io** [comment]
- [x] I will ask our teacher what she means by "explain use of websockets", considering we are not using any websockets that we are aware of :)

Removed websocket from index.html

**Comment by sivert-io** [comment]
- [ ] Jens will push an update regarding a few bugs that were discovered, before we change the hierarchy of the Project.

**Comment by sivert-io** [comment]
- [ ] We also need to lower the animation time on some of the slides (change from 5 seconds to 3 for example)

**Comment by sivert-io** [comment]
I've tried to replace the "long SVG"-elements with the img-tag and added the SVG as a source to the image tag, but that breaks the animations. As far as I know, we cannot access the "layers" inside of the SVG when we are sourcing a .SVG file. That is why we have to keep the "long SVG"-element inside of the index.html

Edit: A solution could be to implement something that adds the inline-code of the SVG-element to the Index.html when the user loads the page. But this seems even worse compared to our current solution. That would make the index.html file smaller, but would be performance heavy if you had to load 15-20 via async.
Here's an example function that could do just what I wrote:
```
<img id="logo" src="logo.svg" />
<script>
// load logo inline for animation on hover
const loadLogo = async () => {
  // get logo, parse source text, and insert text into doc
  document.write(await (await fetch("logo.svg")).text());
  // remove fallback img logo
  document.querySelector("img#logo").remove();
};
loadLogo();
</script>
```
